source/dem/particle_wall_nonlinear_force.cc

```diff
@@ -86,22 +86,19 @@ ParticleWallNonLinearForce<dim>::ParticleWallNonLinearForce(
     }
 
   if (dem_parameters.model_parameters.rolling_resistance_method ==
-      Parameters::Lagrangian::ModelParameters::RollingResistanceMethod::
-        no_resistance)
+      Parameters::Lagrangian::RollingResistanceMethod::no_resistance)
```

Comment:
Why not a template for particle-wall forces too? Not worth it since there not enough contacts?

Reply:
It will be worth it eventually, just in a next PR :).